app/jobs/wakes/update_facebook_metrics_job.rb

```diff
@@ -2,7 +2,7 @@
 class Wakes::UpdateFacebookMetricsJob < ActiveJob::Base
   queue_as :wakes_facebook_metrics
 
-  def perform(locations)
+  def perform(*locations)
```

Comment:
@benhutton I added in a splat to see if it was required for serialization / deserialization to work correctly when a job was enqueued to be 'performed later'. Turns out it is not required. I am curious to know though whether we should keep this around.